app/app.go

```diff
@@ -560,7 +644,9 @@ func New(
 	app.ScopedTransferKeeper = scopedTransferKeeper
 	app.ScopedWasmKeeper = scopedWasmKeeper
 
-	app.registerUpgradeHandlers()
+	if err := app.registerUpgradeHandlers(); err != nil {
+		panic("Failed to register upgradeHandler: " + err.Error())
+	}
```

Comment:
AFAIK, `registerUpgradeHandlers()` might be called before sealing.

Reply:
fixed in [5dbc7e6](https://github.com/medibloc/panacea-core/pull/466/commits/5dbc7e60f4a50044ed9c840d7eee5091a459feb3)

Reply:
Thank you!

Reply:
Danke. I even forgot about it even though we had already handled this issue on the master branch.